sync/rpc/server.go

```diff
@@ -966,6 +981,7 @@ func (s *SyncAgentServer) processRemoveSnapshot(snapshot string) error {
 				return err
 			}
 		}
+		s.PurgeStatus.Progress = int((float32(index)/float32(count) + (float32(opIndex)+1)/float32(opCount)) * 100)
```

Comment:
Unnecessary since the most time-consuming one here is `sfold`.

It's better if we can tap into `sfold` and get some progress from https://github.com/longhorn/sparse-tools/blob/master/sparse/sfold.go#L52 . (need to check `sfold` into function call instead fo binary call)